youtube_dl/extractor/twitch.py

```diff
@@ -358,9 +358,17 @@ def _extract_playlist(self, channel_id):
                 break
             offset += limit
         return self.playlist_result(
-            [self.url_result(entry) for entry in orderedSet(entries)],
+            [self._make_url_result(entry) for entry in orderedSet(entries)],
             channel_id, channel_name)
 
+    def _make_url_result(self, url):
+        try:
+            video_id = 'v%s' % TwitchVodIE._match_id(url)
+            return self.url_result(url, TwitchVodIE.ie_key(), video_id=video_id)
+        except AssertionError:
+            self.to_screen('Unable to match video ID from URL: %s' % url)
```

Comment:
Remove.

Reply:
`_match_id(...)` throws an `AssertionError` if the regex match fails, i.e. if the URL doesn't look like a VOD URL. Would you prefer this error be propagated, or should I match the URL differently? Or are you just objecting to the log line?

Reply:
I'm addressing this concrete line of code.